.github/workflows/ci-image-checks.yml

```
@@ -292,7 +292,7 @@ jobs:
       - name: "Save docs inventory cache"
         uses: apache/infrastructure-actions/stash/save@1c35b5ccf8fba5d4c3fdf25a045ca91aa0cbc468
         with:
-          path: ./docs/_inventory_cache/
+          path: ./generated/generated-docs//_inventory_cache/
```

Uuups, double-slash?
```suggestion
          path: ./generated/generated-docs/_inventory_cache/
```

---

Fixed already :)